### ISSUE TITLE
Wipe&prime pillar is too solid

### ISSUE THREAD

**Issue by TheRuttmeister** [daid/LegacyCura#948]
When printing the wipe and prime pillar (15mm3), its printed solid. If there is any over extrusion at all, and even when there isn't, it gets knocked over by the printing process (100% of the time so far).

I'd suggest that the pillar should have a slightly larger footprint and be less that 100% filled (30% is probably plenty).

**Comment by Dim3nsioneer** [comment]
I recommend the usage of a brim which is wide enough to include both print and wipe tower.

However, if changes are made at the W&P tower, I would like to through my 20 cents into the discussion: I experimented with the shape of the W&P tower and found a rectangular shape to have its advantages if the y dimension is longer than the distance between the two nozzles. In this case both nozzles are wiped clean...

**Comment by daid** [comment]
@Dim3nsioneer The 2nd nozzle is always wiped over the tower then it's finished printing it, so the size should be irrilivant for that bit.

The wipe tower code does need some work, but all dual-extrusion code does. Latest PinkUnicorn engine code already has a bunch of changes that will improve stuff for dual-extrusion.

**Comment by Dim3nsioneer** [comment]
@daid Agreed on the wiping of the second nozzle. However the first nozzle might ooze a bit during the priming time of the second one (looks ugly if one of them is white and the other is not). Increasing the size of the wipe tower to >21.6mm results in wiping the first nozzle too. However, the same effect could be achieved with a rectangular shape using much less material.
An alternative might be to add another wipe movement for the inactive nozzle.

**Comment by TheRuttmeister** [comment]
I've played with a plugin that created a custom size and location wipe tower, and its more trouble than its worth. I just want the wipe tower to not get knocked over. Brim will help, but a sub 100% fill is more important. Seriously, thats all I need for perfect dual colour (or as perfect as things can be before we get better machines).
